src/main/scala/org/broadinstitute/dsde/workbench/sam/openam/PostgresAccessPolicyDAO.scala

```diff
@@ -138,7 +140,44 @@ class PostgresAccessPolicyDAO(protected val dbRef: DbReference,
   }
 
   def deleteResource(resource: FullyQualifiedResourceId): IO[Unit] = ???
-  def loadResourceAuthDomain(resource: FullyQualifiedResourceId): IO[LoadResourceAuthDomainResult] = ???
+
+  def loadResourceAuthDomain(resource: FullyQualifiedResourceId): IO[LoadResourceAuthDomainResult] = {
+    runInTransaction { implicit session =>
+      val ad = AuthDomainTable.syntax("ad")
+      val r = ResourceTable.syntax("r")
+      val rt = ResourceTypeTable.syntax("rt")
+      val g = GroupTable.syntax("g")
+
+      val query = samsql"""select ${g.result.name}
+              from ${ResourceTable as r}
+              join ${ResourceTypeTable as rt} on ${r.resourceTypeId} = ${rt.id}
+              left outer join ${AuthDomainTable as ad} on ${r.id} = ${ad.resourceId}
+              left outer join ${GroupTable as g} on  ${ad.groupId} = ${g.id}
```

Comment:
It took me a little while to figure out why you were doing the outer joins here.  I think the query would work fine with regular inner joins, but then if there were no results, we wouldn't be able to tell if it was because the resource didn't exist or if it was because the resource is just not constrained.  Is that right?  If so, can we put a brief comment here that calls this out, because I don't think it's immediately obvious.  Good job finding a way to get it all done in one query though.